README.adoc

```diff
@@ -28,30 +28,26 @@ Instead, there is a reference to an abstract _target_: `search_engine`.
 
 This reference allows the same scenario to be executed against different environments.
 
-== Creating another environment
+== Using multiple environment
 
 * Open the link:src/main/kotlin/com/chutneytesting/hello/Environments.kt[Environments.kt] file.
 
-* Duplicate the existing `google_fr` and `environment_fr` values as `google_en` and `environment_en`
+* Look at the  `google_fr` and `environment_fr` values and  `google_en` and `environment_en` values. They share the same name: `"search_engine"`
 
-* Now replace the website extension by `.com` instead of `.fr`
-
-Newly created code should look like
+* With the same scenario, you can execute the http get task on target `search_engine` on two different environments
 
 [source,kotlin]
 ----
-val google_en = ChutneyTarget(name = "search_engine", url = "https://www.google.com")
-
-val environment_en = ChutneyEnvironment(
-    name = "The English World Wide Web",
-    description = "The World Wide Web, mostly",
-    targets = listOf(
-        google_en
-    )
+ HttpGetTask(
+    target = "search_engine",
+    uri = "/",
+    validations = mapOf("request accepted" to "status == 200".spEL()),
+    outputs = mapOf("body" to "body".spEL())
```

Comment:
Even for example purpose, is this really necessary ? Or maybe we can change the output key ?